doc/guide/admin.rst

```diff
@@ -852,6 +848,272 @@ to be added to one or more PAM stacks, e.g.
+
+.. code-block:: console
+
+    $ sudo systemctl start flux
+    $ flux uptime
+     07:42:52 run 3.8s,  owner flux,  depth 0,  size 8,  7 offline
+    $ systemctl status flux
+      ● flux.service - Flux message broker
+       Loaded: loaded (/lib/systemd/system/flux.service; enabled; vendor preset: enabled)
+       Active: active (running) since Tue 2024-04-23 07:36:44 PDT; 37s ago
+      Process: 287736 ExecStartPre=/usr/bin/loginctl enable-linger flux (code=exited, status=0/SUCCESS)
+      Process: 287737 ExecStartPre=bash -c systemctl start user@$(id -u flux).service (code=exited, status=0/SUCCESS)
+     Main PID: 287739 (flux-broker-0)
+       Status: "Running as leader of 8 node Flux instance"
+        Tasks: 22 (limit: 8755)
+       Memory: 26.6M
+          CPU: 3.506s
+       CGroup: /system.slice/flux.service
+               └─287739 broker --config-path=/etc/flux/system/conf.d -Scron.directory=/etc/flux/system/cron.d -Srundir=/run/flux -Sstatedir=/var/lib/flux -Slocal-uri=local:///run/flux/local -Slog-stderr-level=6 -Slog-stderr-mode=local -Sbroker.rc2_none -Sbroker.quorum=1 -Sbroker.quorum-timeout=none -Sbroker.exit-norestart=42 -Sbroker.sd-notify=1 -Scontent.dump=auto -Scontent.restore=auto
+
+  Apr 23 07:36:46 test0 flux[287739]: sched-fluxion-resource.info[0]: version 0.33.1-40-g24255b38
+  Apr 23 07:36:46 test0 flux[287739]: sched-fluxion-qmanager.info[0]: version 0.33.1-40-g24255b38
+  Apr 23 07:36:46 test0 flux[287739]: broker.info[0]: rc1.0: running /etc/flux/rc1.d/02-cron
+  Apr 23 07:36:47 test0 flux[287739]: broker.info[0]: rc1.0: /etc/flux/rc1 Exited (rc=0) 2.6s
+  Apr 23 07:36:47 test0 flux[287739]: broker.info[0]: rc1-success: init->quorum 2.65475s
+  Apr 23 07:36:47 test0 flux[287739]: broker.info[0]: online: test0 (ranks 0)
+  Apr 23 07:36:47 test0 flux[287739]: broker.info[0]: quorum-full: quorum->run 0.102056s
+
+Do other nodes join?
+====================
+
+Bring up a follower node that is configured with the leader as its parent
+in the tree based overlay network:
+
+.. code-block:: console
+
+  $ ssh test1
+  $ sudo systemctl start flux
+  $ flux uptime
+   07:47:58 run 4.3m,  owner flux,  depth 0,  size 8,  6 offline
+  $ flux overlay status
+  0 test0: partial
+  ├─ 1 test1: partial
+  │  ├─ 3 test3: offline
+  │  ├─ 4 test4: offline
+  │  └─ 5 test5: offline
+  └─ 2 test2: offline
+     ├─ 6 test6: offline
+     └─ 7 test7: offline
+  $ flux resource list
+       STATE UP NNODES NODELIST
+     avail  ✔      2 test[0-1]
+    avail*  ✗      6 test[2-7]
+
+Are my queues started?
+======================
+
+If named queues are configured, they will be initially stopped, meaning
+jobs can be submitted but won't run.  Enable all queues with
+
+.. code-block:: console
+
+  $ sudo flux queue start --all
+  debug: Scheduling is started
+  batch: Scheduling is started
+
+Can I run a job?
+================
+
+.. code-block:: console
+
+  $ flux run hostname
+  test1
```

Comment:
Perhaps at this point it would be useful to validate that the job runs as the correct user, with supplementary groups, etc, e.g.:

```console
$ id
uid=1000(pi) gid=1000(pi) groups=1000(pi),27(sudo),114(netdev)
$ flux run -N4 id
uid=1000(pi) gid=1000(pi) groups=1000(pi),27(sudo),117(netdev)
uid=1000(pi) gid=1000(pi) groups=1000(pi),27(sudo),114(netdev)
uid=1000(pi) gid=1000(pi) groups=1000(pi),27(sudo),117(netdev)
uid=1000(pi) gid=1000(pi) groups=1000(pi),27(sudo),114(netdev)
```